query-node/substrate-query-framework/cli/src/templates/entities/model.ts.mst

```diff
@@ -91,5 +102,16 @@ export {{#isInterface}}abstract{{/isInterface}} class {{className}}
       {{#description}}description: `{{{description}}}`{{/description}} })
     {{camelName}}{{^required}}?{{/required}}{{#required}}!{{/required}}:{{tsType}} 
   {{/is.enum}}
+
+
+  {{#is.union}} 
+	@Column('jsonb')
```

Comment:
Could you use `@CustomField({ db: { type: 'jsonb' }, api: { type: 'json' } })` ?

Reply:
Based on what I have researched `api: { type: 'json' }` would make the resulting (scalar) GraphQL type `JSON`, while here we preserve more structure by making it a union type

Reply:
So the query _input_ field is JSON, while the result is of Union type and strongly typed

Reply:
Okay! Got it!